### ISSUE TITLE
Suggestion: Add overall progress indicator when downloading mods

### ISSUE THREAD

**Issue by nhansendev** [cdbbnnyCode/modpack-installer#17]
This is just a small QoL suggestion:

When downloading large modpacks there are dozens to hundreds of mods to download, but no way to tell how close to completion the process is. It would be nice to have something like `f"[{current_index}/{total_mod_count}]"` printed inline with the existing info to give a hint at the progress.

**Comment by cdbbnnyCode** [comment]
Good idea. This should be very simple since the downloads aren't executed in parallel anymore (there used to be 4 download threads to speed up the process, but I'm trying to ratelimit it now for the sake of keeping the API key).
